cico_setup.sh

```diff
@@ -47,3 +46,8 @@ function cico_setup() {
   install_deps;
   prepare;
 }
+
+function run_tests_without_coverage() {
```

Comment:
why do you need this in the cico setup?

Reply:
there are some tests [link](https://github.com/fabric8-services/fabric8-cluster-client/blob/254ee6bb23168f9edfcbb6c929001b7c0dc1f1f6/service/service_blackbox_test.go) to run these tests during build, we need this.  any other concern ??

Reply:
ah never mind, I guess we need both test WITH and WITHOUT coverage actually (the `_without_coverage` part was my main of concern)